core/src/main/scala/ai/h2o/sparkling/backend/converters/SparkDataFrameConverter.scala

```diff
@@ -64,8 +66,9 @@ object SparkDataFrameConverter extends Logging {
 
   def toDataFrame(hc: H2OContext, fr: ai.h2o.sparkling.frame.H2OFrame, copyMetadata: Boolean): DataFrame = {
     // Relation referencing H2OFrame
-    val relation = ExternalBackendH2OFrameRelation(fr, copyMetadata)(hc.sparkSession.sqlContext)
-    hc.sparkSession.sqlContext.baseRelationToDataFrame(relation)
+    val spark = SparkSessionUtils.active
+    val relation = ExternalBackendH2OFrameRelation(fr, copyMetadata)(spark.sqlContext)
+    spark.sqlContext.baseRelationToDataFrame(relation)
```

Comment:
Can't we just use `spark.baseRelationToDataFrame(relation)`?

Reply:
👍